src/types.ts

```diff
@@ -1,4 +1,5 @@
 export interface SelectOption {
 	label: string
 	value: string | string[]
+	$isDisabled?: boolean
```

Comment:
Is there a particular reason to add the `$` at the beginning here?

Reply:
I made same way how it was in vue-multiselect, we can easily get rid from $ and call property as we want.

Reply:
@hungsu what do you think about naming disabled fields? just isDisabled?